<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DragDropClass</class>
 <widget class="QMainWindow" name="DragDropClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>841</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>DragDrop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DropArea" name="droparea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>DropArea</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
          </font>
         </property>
         <property name="text">
          <string>文件类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="FileTypeText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
          </font>
         </property>
         <property name="text">
          <string>文件名称</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="FileNameText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
          </font>
         </property>
         <property name="text">
          <string>文件路径</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="FilePathText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DropArea</class>
   <extends>QLabel</extends>
   <header>DropArea.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DragDrop.qrc"/>
 </resources>
 <connections/>
</ui>
